edge/client/portainer_edge_client.go

```diff
@@ -251,3 +251,7 @@ func (client *PortainerEdgeClient) ProcessAsyncCommands() error {
 func (client *PortainerEdgeClient) SetLastCommandTimestamp(timestamp time.Time) {
 	return // edge mode only
 }
+
+func (client *PortainerEdgeClient) EnqueueLogCollectionForStack(logCmd LogCommandData) error {
+	return nil
```

Comment:
is there a reason why we don't enable that for a regular edge?

Reply:
We could add it for the regular edge too, but it would have to be a mostly completely new implementation.